dgraphtest/config.go

```diff
@@ -31,39 +31,39 @@ type UpgradeCombo struct {
 
 func AllUpgradeCombos(v20 bool) []UpgradeCombo {
 	fixedVersionCombos := []UpgradeCombo{
-		// OPEN SOURCE RELEASES
-		{"v21.03.0", "v23.0.1", BackupRestore},
-		{"v21.03.0-92-g0c9f60156", "v23.0.1", BackupRestore},
-		{"v21.03.0-98-g19f71a78a-slash", "v23.0.1", BackupRestore},
-		{"v21.03.0-99-g4a03c144a-slash", "v23.0.1", BackupRestore},
-		{"v21.03.1", "v23.0.1", BackupRestore},
-		{"v21.03.2", "v23.0.1", BackupRestore},
-		{"v22.0.0", "v23.0.1", BackupRestore},
-		{"v22.0.1", "v23.0.1", BackupRestore},
-		{"v22.0.2", "v23.0.1", BackupRestore},
+		// OPEN SOURCE RELEASES, 4fc9cfd => 23.1.0
```

Comment:
I'd like to understand why we need to refer to the hash and not the release version. Also if you could add that comment in the code too, that'd be great.

Reply:
The upgrade test fails for v23.1 because of a recent change that was made in an error. We will move it a newer release as soon as we have another.

Reply:
Added a comment.